src/ch/imedias/rsccfx/view/RsccPresenter.java

```diff
@@ -6,7 +6,10 @@
   private final Rscc model;
   private final RsccView view;
 
-  /** Javadoc comment here. */
+  // FIXME: Use this as main Presenter and base all of the presenters around it on this
```

Comment:
@ everyone Use this as main Presenter and base all of the presenters around it on this